apps/hpl/2.1/8proc_hpl_example.sh.tpl

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash -l
+#@      cw_TEMPLATE[name]="HPL v2.1 Template (GridScheduler)"
```

Comment:
I'd call this "HPL v2.1 - 8 processes (GridScheduler)".  It's not really a "template" to the end user.